superset-frontend/src/SqlLab/main.less

```
@@ -158,6 +158,7 @@ div.Workspace {
 
       &:last-child {
         margin-right: 0;
+        font-family: @font-family-monospace;
```

```suggestion
        font-feature-settings: 'kern' 1, 'tnum' 1;
```
^ this is another option, if you want to stick with the `Inter UI` font rather than changing to `Fira Code`. This turns on the "tabular numbers" feature, which should have the same fixed-width effect.